clients/ui/frontend/src/app/pages/modelCatalogSettings/screens/CatalogSourceConfigsTableRow.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { ActionsColumn, Td, Tr } from '@patternfly/react-table';
+import { Button, Label, Switch } from '@patternfly/react-core';
+import { useNavigate } from 'react-router-dom';
+import { CatalogSourceConfig } from '~/app/modelCatalogTypes';
+import { manageSourceUrl } from '~/app/routes/modelCatalogSettings/modelCatalogSettings';
+import { useNotification } from '~/app/hooks/useNotification';
+import {
+  CATALOG_SOURCE_TYPE_LABELS,
+  ModelVisibilityBadgeColor,
+} from '~/concepts/modelCatalogSettings/const';
+import { hasSourceFilters, getOrganizationDisplay } from '~/concepts/modelCatalogSettings/utils';
+
+type CatalogSourceConfigsTableRowProps = {
+  catalogSourceConfig: CatalogSourceConfig;
+  onDelete?: (config: CatalogSourceConfig) => void;
+};
+
+const CatalogSourceConfigsTableRow: React.FC<CatalogSourceConfigsTableRowProps> = ({
+  catalogSourceConfig,
+  onDelete,
+}) => {
+  const navigate = useNavigate();
+  const notification = useNotification();
+  const isDefault = catalogSourceConfig.isDefault ?? false;
+  const isEnabled = catalogSourceConfig.enabled ?? true;
+
+  const hasFilters = React.useMemo(
+    () => hasSourceFilters(catalogSourceConfig),
+    [catalogSourceConfig],
+  );
+
+  const handleEnableToggle = (checked: boolean) => {
+    // For now, just show a notification as per requirements
+    // TODO: RHOAIENG-38347 - Implement actual enable/disable functionality
```

Comment:
We should not reference RH Jira tickets upstream. Can you remove these RHOAIENG comments and just leave generic TODOs?